src/client/client.c

```diff
@@ -2448,7 +2448,7 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
                 }
                 else if (XSTRNCMP(myoptarg, "verifyInfo", 10) == 0) {
                     printf("Verify should not override error\n");
-                    myVerifyAction = VERIFY_USE_PREVERFIY;
+                    myVerifyAction = VERIFY_USE_PREVERIFY;
```

Comment:
@JacobBarthelmeh  Should this code rely on wolfssl/test.h?

Reply:
It should be separated at some point because as wolfCLU server/client gets more features it will likely diverge away from it.